src/main/modules/oidc/index.ts

```diff
@@ -86,9 +86,9 @@ export const idamCallbackHandler = async (
   }
 
   //For now if user account does not have the citizen role redirect to login
-  if (!req.session.user?.isCitizen) {
-    return res.redirect(AuthUrls.LOGIN);
-  }
+  // if (!req.session.user?.isCitizen) {
```

Comment:
also this shouldn't be committed

Reply:
accidentally committed and has been reverted